CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- 14 new fonts
+	1. smallcaps3
+	2. fraktur2
+	3. bold fraktur
+	4. bold script
```

Comment:
`bold script` or `bold_script` ?
Please use exactly the font name in `CHANGELOG.md` ;-)

Reply:
all-right i will